test/cases/ostree-ng.sh

```diff
@@ -13,6 +13,9 @@ function greenprint {
     echo -e "\033[1;32m${1}\033[0m"
 }
 
+# Install skopeo 
+sudo dnf install -y skopeo
```

Comment:
I think we should add this to osbuild-composer.spec, requirements for the -tests package.

Reply:
Thank you. Added skopeo in osbuild-composer.spec - requirements.